src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -35,28 +36,31 @@ $-control-size: 32px;
 
 .VideoHuddleControls {
   @include sparkle.panel--normal;
+  width: 5%;
   position: relative;
   margin-top: space.gap(-1);
   display: flex;
   flex-direction: column;
   justify-self: start;
   border-left: 1px solid theme.$sparkle--border;
 
-  span {
-    border-bottom: 1px solid theme.$sparkle--border;
-    width: $-control-size;
-    height: $-control-size;
-    opacity: 0.5;
+  @media (max-width: breakpoint.$lg) {
+    width: 10%;
```

Comment:
10% ? looks a bit hacky.
I don't think we need here a different width depending on the different breakpoints. 
you can also use here a scss variable.